zerver/lib/api_test_helpers.py

```diff
@@ -204,6 +204,21 @@ def get_streams(client):
     validate_against_openapi_schema(result, '/streams', 'get', '200')
     assert len(result['streams']) == 4
 
+def get_user_groups(client):
+    # type: (Client) -> None
+
+    # {code_example|start}
+    # Get all user groups of the realm
+    result = client.get_user_groups()
+    # {code_example|end}
+
+    validate_against_openapi_schema(result, '/user_groups', 'get', '200')
+    user_groups = [u for u in result['user_groups'] if u['name'] == "other users"]
+    assert user_groups[0]['description'] == 'Other users'
+
+    validate_against_openapi_schema(result, '/user_groups', 'get', '200')
+    assert len(result['streams']) == 2
```

Comment:
I guess these two lines popped up after a copy-paste and shouldn't be here anymore.